flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -487,6 +491,12 @@ private void callSet(SqlCommandCall cmdCall) {
         }
     }
 
+    private void setSqlCliVerbose(String key, String value) {
+        if (SqlClientOptions.VERBOSE.key().equals(key)) {
+            isVerbose = Boolean.parseBoolean(value);
+        }
+    }
```

Comment:
It's a little hack. I think we'd better to read option value from the SessionContext. 

Because we also have RESET command and it would be modified by the RESET.